bigquery/simpleapp/simpleapp.go

```diff
@@ -46,17 +56,8 @@ func main() {
 	}
 }
 
-// query returns a slice of the results of a query.
-func query(proj string) (*bigquery.RowIterator, error) {
-	// [START bigquery_simple_app_client]
-	ctx := context.Background()
-
-	client, err := bigquery.NewClient(ctx, proj)
-	if err != nil {
-		return nil, err
-	}
-	defer client.Close()
-	// [END bigquery_simple_app_client]
+// query returns a row iterator suitable for reading query results.
+func query(ctx context.Context, client *bigquery.Client) (*bigquery.RowIterator, error) {
 
 	// [START bigquery_simple_app_query]
```

Comment:
Might want to include the function declaration in the sample? Up to you.

Reply:
I'd defer that until we have a chance to address all languages, since they all use this sparse style: https://cloud.google.com/bigquery/docs/quickstarts/quickstart-client-libraries#bigquery_simple_app_client-go